runtime/compiler/codegen/J9TreeEvaluator.cpp

```diff
@@ -36,27 +36,42 @@
 #include "runtime/J9ValueProfiler.hpp"
 #include "util_api.h"
 
+
+TR::Snippet *
+J9::TreeEvaluator::getFieldWatchInstanceSnippet(TR::CodeGenerator *cg, TR::Node *node, J9Method *m, UDATA loc, UDATA os)
+   {
+   return new(cg->trHeapMemory()) TR::J9WatchedInstanceFieldSnippet(cg, node, m, loc, os);
```

Comment:
To stick with convention, can we add a space between `new` and `(`? Similarly below.